.gitignore

```diff
@@ -4,3 +4,5 @@ tmp
 .secrets
 sites/*
 !sites/.gitkeep
+services/storage/*.crt
+services/storage/*.key
```

Comment:
Why `*.key`? Cert is generated in startup, but keys are not.

Reply:
My bad, they are. However we already use .key files for storage nodes. I suggest you to ignore keys with tls postfix.

Reply:
on `make up` i had two files generated locally: `s04tls.crt` and `s04tls.key`

i guess it is generated with `-keyout` flag [here](https://github.com/nspcc-dev/neofs-dev-env/blob/master/services/storage/artifacts.mk#L16) and used [here](https://github.com/nspcc-dev/neofs-dev-env/blob/master/services/storage/docker-compose.yml#L121)

Reply:
oh, comments were not updated

so you mean change it to `*tls.key`?

Reply:
> so you mean change it to *tls.key?

Yes, it will not affect storage node keys this way.

Reply:
in fact, i hastened to exclude all keys and did not notice that there were more keys in that dir, thank you

changed to `*tls.*` both for cert and key

Reply:
in fact, i hastened to exclude all keys and did not notice that there were more keys in that dir, thank you

changed to `*tls.*` both for cert and key